components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -447,6 +466,46 @@ - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentConta
   [self.view setNeedsLayout];
 }
 
+- (void)expandToFullScreenWithDuration:(NSTimeInterval)duration {
```

Comment:
is the below calculation really needed or do you think it is similar to other calculations we have already made? just making sure we aren't re-calculating twice